src/internal.rs

```diff
@@ -14,6 +14,12 @@ macro_rules! __declare_internal_bitflags {
         $iter_vis:vis struct $Iter:ident;
         $iter_names_vis:vis struct $IterNames:ident;
     ) => {
+        #[cfg(feature = "zerocopy_0_6")]
+        use $crate::__private::zerocopy;
+        #[cfg(feature = "zerocopy_0_6")]
+        use ::core::marker::Sized;
+
+        #[cfg_attr(feature = "zerocopy_0_6", derive($crate::__private::zerocopy::AsBytes, $crate::__private::zerocopy::FromBytes))]
```

Comment:
Unfortunately this won't actually work. This code is expanded in an end-user's library where there's no knowledge of whether or not `bitflags` has this feature enabled. For other libraries, we instead expand to a macro that calls back into `bitflags` and generates appropriate `impl` blocks.

Since this library requires using their custom derive I think we'll need a different strategy that accepts the struct definition as input and annotates it with its attributes. We'll need a second macro that also includes those necessary `use` statements.

If `zerocopy` `0.7` that's currently in-progress has a nicer experience for implementing its traits then I think it would be good to wait for it.

Reply:
Hey, came across this while discussing https://github.com/google/zerocopy/issues/287. Zerocopy 0.7 has been released now. Is the experience you're referring to the need to emit an impl for the function `only_derive_is_allowed_to_implement_this_trait` or something else? We've recently done some work to [make the docs more precise](https://github.com/google/zerocopy/pull/283) with regards to what needs to hold in order for a trait impl to be sound, so that might help you. Let me know if there's anything else we could add that would make this easier for you!

Reply:
@qwandor I believe I've figured out a way to support this: https://github.com/google/zerocopy/issues/1497#issue-2393919599

If you're still interested in zerocopy support and have some cycles to donate, let me know. It'd involve adding support for a `#[zerocopy(crate = ...)]` attribute (https://github.com/google/zerocopy/issues/11), which we currently don't have cycles for on our side.

Reply:
I think #348 works for everything I need, so no explicit zerocopy support is needed here.